public_html/wp-content/themes/pattern-directory/src/components/pattern-preview-actions/copy-guide.js

```diff
@@ -0,0 +1,81 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { Animate, Guide } from '@wordpress/components';
+import { createInterpolateElement } from '@wordpress/element';
+
+const CopyPasteImage = () => (
+	// Wrap the image to avoid the UI shift after the GIF loads
+	<div style={ { height: '220px' } }>
+		<img
+			src="/wp-content/themes/pattern-directory/images/copy-paste-demo.gif"
+			alt={ __( 'GIF of copy and pasting.', 'wporg-patterns' ) }
+		/>
+	</div>
+);
+
+const CopyGuide = ( { onFinish } ) => {
+	return (
+		<Animate type="appear" options={ { origin: 'bottom' } }>
+			{ ( { className } ) => (
+				<Guide
+					className={ `pattern-actions__guide ${ className }` }
+					onFinish={ onFinish }
+					finishButtonText={ __( 'Close', 'wporg-patterns' ) }
```

Comment:
Is this a required prop, or can we get rid of this extra close button? There are a lot of other ways to close this modal, so this feels redundant.

Reply:
I made use of the `Guide` component because it's structurally the same as the design and it's quick to get up and running. However, since it does expect a sequence of user interactions, there is always a button on the last page. https://github.com/WordPress/gutenberg/blob/4ae96fb2867b28a5c8ecb2847e74f99c9904a909/packages/components/src/guide/index.js#L46. The `finishButtonText` allows you to override the default text which is by default `Finish`.

We could alternately build our own modal, but of me, the extra button to close is not a deal-breaker and does help mouse-driven users... who could potentially have a hard time finding the "x" button overlayed on the image.